contrib/runners/python_runner/python_runner.py

```diff
@@ -105,7 +109,8 @@ def pre_run(self):
     def run(self, action_parameters):
         LOG.debug('Running pythonrunner.')
         LOG.debug('Getting pack name.')
-        pack = self.get_pack_name()
+        pack = self.get_pack_ref()
+        pack_db = Pack.get_by_ref(pack)
```

Comment:
I guess there could be some races now - e.g. when pack is being removed and the model is removed first, but then again, we can't really guarantee proper behavior during pack removal process anyway.